### DISCUSSION TITLE
Feature Request: 2 Layers of Horizontal Buttons Stack

### DISCUSSION THREAD

**Discussion by xyeoda** [Clooos/Bubble-Card#308] [Feature requests]
I have 3 floors within my house. If i could have Buttons Stack in 3 layers based on Floors then it would be much easier to navigate without having to go to another page per floor.

**Comment by Clooos** [comment]
Adding more than one horizontal buttons stack is in my todo list since the first beta, but you can probably get around of that limitation with a custom style by changing the `bottom` propriety. But I haven't tried that yet.

**Reply by Breakingx** [reply]
Hey! Just wanted to add, that I'm interested in this feature, too! :)
